.circleci/config.yml.in

```diff
@@ -359,6 +359,33 @@ jobs:
       - run_tests_selective:
           file_or_dir: test/test_prototype_*.py
 
+  unittest_jpeg_ref:
```

Comment:
Maybe we should add a comment here, indicating why we do this separately - something similar to the comment/skip message you wrote in the test?